mscore/plugin/pluginUpdater.cpp

```diff
@@ -552,6 +552,7 @@ void ResourceManager::uninstallPluginPackage()
 static const std::map<PluginStatus, PluginButtonStatus> buttonStatuses = {
       {NOT_INSTALLED,{QObject::tr("Install"),true,false}},
       {INSTALL_FAILED,{QObject::tr("Install failed. Try again"),true,false}},
+      {INSTALL_FAILED_INVALID,{QObject::tr("Install failed. Invalid plugin."),false,false}},
```

Comment:
Can you plesdse insert some spaces here? Like `{INSTALL_FAILED_INVALID,{QObject::tr("Install failed. Invalid plugin."), false, false}},`, here in in the other initializers please

Reply:
OK.